### DISCUSSION TITLE
PBs for funboxes

### DISCUSSION THREAD

**Discussion by tricarbonate** [monkeytypegame/monkeytype#1686] [Questions] [answered]
I've been practising numbers and special characters with funboxes, and I noticed that the system doesn't keep track of PBs for these modes. Is this on purpose? Or has it been overlooked?

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I don't think there's a point in having seperate pb's for funboxes. The point of funboxes is to have "fun" not like a challenge where you have to type as fast as possible. Also you would have to store an extra 20 or so pbs. I could understand why a pb would exist for something like "weak spot" but I couldn't see it for anything else.
